OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Interview mit Marie Saulnier Bloch
+collection: 'Einbürgerungen à la «Schweizermacher»?'
+protagonist: Marie Saulnier Bloch
+creator: Amanda Unger
+birthday: 26.05.1974
```

Comment:
this shouldn't be here after the change in the md-generation-file

Reply:
same for above, I deleted all except creator